test.rb

```diff
@@ -0,0 +1,18 @@
+a = "bbb"
+
+def foo
+    'a'
```

Comment:
__[rubocop]__
* Use 2 (not 4) spaces for indentation. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-indentation)